frontend/src/components/TriggeredPrompt.tsx

```diff
@@ -3,6 +3,9 @@ import React, {
   ReactElement,
   useEffect,
   useState,
+  useMemo,
```

Comment:
useMemo is never used -- ok to remove then?

Reply:
ugh i'm so used to eslint handling everything for me 😩